core/src/main/java/org/mule/processor/AsyncInterceptingMessageProcessor.java

```diff
@@ -72,7 +72,7 @@ public WorkManager getWorkManager() throws MuleException
 
     public void start() throws MuleException
     {
-        if (workManager != null)
+        if (workManager != null && !workManager.isStarted())
```

Comment:
Well.. that seems a bug that must be fixed, because the workMananger is created on this class constructor and the lifecycle is applied (or should be applied) only from inside this class